1. CWE Base Mod/Cold War Era Mod (CWE)/common/history/characters/can - canada.txt

```diff
@@ -0,0 +1,35 @@
+CHARACTERS = {
+	c:CAN = {
+		
+		create_character = {
+			first_name = "louis_st_laurent_first_name"
+			last_name = "louis_st_laurent_last_name"
+			ruler = yes
+            ig_leader = yes
+			birth_date = 1882.2.1
+			interest_group = ig_professionals 
+			ideology = ideology_liberal
```

Comment:
This needs to be a character ideology

Reply:
![image](https://user-images.githubusercontent.com/34218031/229434152-3a2b1643-182e-48e6-8c25-0ccf9f8eb7dd.png)
This is a character ideology.